### ISSUE TITLE
[Feature Request] Hide labels inside of bundles.

### ISSUE THREAD

**Issue by GollyJer** [leggett/simplify#703]
In line with the Simplify philosophy, it would be awesome to have the ability to hide the redundant labels inside bundles.

![image](https://user-images.githubusercontent.com/689204/178849291-19a245cd-80c9-4fc5-8d7c-941f41becf18.png)

**Comment by leggett** [comment]
I do have an option that moves the labels to the right. 

Hiding them is a bit harder and I worry it is confusing for when there are a lot of messages in the bundle.

**Comment by GollyJer** [comment]
Fair enough... In my head Simplify is named "Context Inbox" and each label is a context I can process quickly. I never even really thought about the case of threads having multiple labels. 

I suppose the ideal implementation would only hide the bundles' label but leave the others visible.

Feel free to close this issue if it's not something you're interested in. 👍

**Comment by leggett** [comment]
Ok, I looked into this some more. Some thoughts:

- It isn't too hard to hide just the label for the bundle
- I have a history of being in the minority opinion on this issue (for a time, Gmail showed the label on every message when you opened a label → that was my call and eventually due to feedback, it was switched back to hiding the label)
- I think this at least be an option with the default set to *hide* the label
- I think there are a couple other things I should do to make it a good default (maybe make the bundle title bar sticky or at least limit how many messages you can see at first when you expand a bundle -- Inbox would only show up to 25)

So I'll do this. Not sure how much I will do before I ship it, but I will do it.

**Comment by GollyJer** [comment]
It's fun to talk with someone who's been on the inside! Simply bringing up this feature request has made me aware of Gmail UI choices a little more lately.

- Gmail hides the active label when viewing by label but keeps the others. It seems like a sensible default for bundles.
- It's interesting that Gmail desktop puts the labels at the front of the message, but Gmail app puts them at the end. I love that you have the option to move them to the end. It's more sensible and provides consistency.
- I like the idea of a sticky header vs. limiting the message count. One less click. Scrolling is easy.

No hurry here. I just wanted to get the idea out there. 🌱
